styleguide/source/_patterns/01-molecules/06-components/10-hub-hero/_hub-hero.scss

```diff
@@ -83,3 +84,53 @@
     margin-top: 0;
   }
 }
+
+// side variant
+%hub-hero-side,
+.hub-hero-side {
+  align-items: stretch;
+
+  .hub-hero-inner {
+    background-color: rgba($white, .45);
+    width: 100%;
+    margin: 0;
+    margin-right: auto;
+    text-align: left;
+    padding: 20px;
+
+    @include breakpoint($bp-small) {
+      width: 70%;
+    }
+
+    @include breakpoint($bp-med) {
+      width: 50%;
+    }
+  }
+
+  .hub-hero_title {
+    background-color: transparent;
+  }
+
+  .hub-hero_desc {
+    background-color: transparent;
+  }
+
+  .hub-hero_button-container {
+    width: 100%;
+    padding: 20px;
+    margin-top: auto;
+
+    .button-small {
+      width: 100%;
+    }
+  }
+}
+
+.hub-hero-side-left {
```

Comment:
Seems like Side Left actually aligns the content right... Is this right? How would you explain it?